torch/nn/modules/dropout.py

```diff
@@ -54,8 +55,10 @@ def forward(self, input):
 
 
 class Dropout2d(_DropoutNd):
-    r"""Randomly zeroes whole channels of the input tensor.
-    The channels to zero-out are randomized on every forward call.
+    r"""Randomly zero-out entire channels (a channel is a 2D feature map of
```

Comment:
I feel that the explanation of channel here isn't very clear still. People who don't know the concept will still be confused how this works. Maybe say something like `` (e.g., the `j`-th channel of the `i`-th sample in the batched input is the 2D tensor `input[i, j]`) ``